src/main/java/core/basesyntax/figures/Rectangle.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax.figures;
+
+import java.util.Locale;
+
+public class Rectangle extends Figure {
+    private double topSide;
+    private double lateralSide;
+    private double area;
+
+    public Rectangle(String color, double topSide, double lateralSide) {
+        super(color);
+        this.topSide = topSide;
+        this.lateralSide = lateralSide;
+        this.area = calculateArea();
```

Comment:
You should not calculate the area in the constructor. Remove this line as the area can be calculated when needed in the getArea() method.